tests/integration/components/bs-accordion-test.js

```diff
@@ -26,14 +26,32 @@ module('Integration | Component | bs-accordion', function (hooks) {
   test('accordion has correct default markup', async function (assert) {
     await render(hbs`
       <BsAccordion as |acc|>
-        <acc.item @value={{1}} @title="TITLE1">CONTENT1</acc.item>
+        <acc.item @value={{1}} @title="TITLE1" data-test-item="1">CONTENT1</acc.item>
         <acc.item @value={{2}} @title="TITLE2">CONTENT2</acc.item>
       </BsAccordion>
     `);
     assert.dom(`.${accordionClass()}`).exists({ count: 1 }, 'accordion has correct class');
+    assert
+      .dom(`.${accordionClass()} .accordion-collapse`)
+      .exists({ count: 2 }, 'collapsable area of accordion has correct class');
     assert
       .dom(`.${accordionClass()} .${accordionItemClass()}`)
       .exists({ count: 2 }, 'accordion item has correct class');
+    assert.dom('[data-test-item="1"] h2').hasText('TITLE1', 'renders title');
+    assert
+      .dom('[data-test-item="1"] .accordion-collapse')
+      .hasAria(
+        'describedby',
+        find('[data-test-item="1"] h2').id,
```

Comment:
This is making the BS4 tests fail, as you are asserting for `h2`, while _current_ BS5 markup has `h5`. I am pretty sure there was a reason for `h2`, maybe it was like this in 4.0 and they changed their examples later?

Anyway given that we are doing a major release soon, I think we can unify this to `h2` everywhere, and mark this PR for extra safety as breaking, what do you think?